docs/en/stack/ml/df-analytics/flightdata-classification.asciidoc

```diff
@@ -0,0 +1,477 @@
+      }
+    }
+  },
+ "evaluation": {
+   "classification": {
+     "actual_field": "FlightDelay",  <3>
+     "predicted_field": "ml.FlightDelay_prediction",  <4>
+     "metrics": {  
+       "multiclass_confusion_matrix" : {}
+     }
+   }
+ }
+}
+--------------------------------------------------
+// TEST[skip:TBD]
+
+<1> The destination index which is the output of the analysis job.
+<2> We calculate the training error by only evaluating the training data.
+<3> The field that contains the ground truth label.
+<4> The field that contains the predicted value.
+
+Next, we calculate the generalization error that represents how well the model 
+performed on previously unseen data:
+
+[source,console]
+--------------------------------------------------
+POST _ml/data_frame/_evaluate
+{
+ "index": "df-flight-delayed",
+   "query": {
+    "term": {
+      "ml.is_training": {
+        "value": false  <1>
+      }
+    }
+  },
+ "evaluation": {
+   "classification": {
+     "actual_field": "FlightDelay",
+     "predicted_field": "ml.FlightDelay_prediction",
+     "metrics": {  
+       "multiclass_confusion_matrix" : {}
+     }
+   }
+ }
+}
+--------------------------------------------------
+// TEST[skip:TBD]
+
+<1> By only evaluating the data that was not used in training, we can 
+calculate the generalization error which shows the algorithm accuracy in making 
+predictions for previously unseen data.
+
+The API call returns the following response:
+
+[source,console-result]
+--------------------------------------------------
+{
+  "classification" : {
+    "multiclass_confusion_matrix" : {
+      "confusion_matrix" : [
+        {
+          "actual_class" : "false", <1>
+          "actual_class_doc_count" : 912, <2>
+          "predicted_classes" : [
+            {
+              "predicted_class" : "false", <3>
+              "count" : 755 <4>
+            },
+            {
+              "predicted_class" : "true",
+              "count" : 157
+            }
+          ],
+          "other_predicted_class_doc_count" : 0
+        },
+        {
+          "actual_class" : "true",
+          "actual_class_doc_count" : 306,
+          "predicted_classes" : [
+            {
+              "predicted_class" : "false",
+              "count" : 80
+            },
+            {
+              "predicted_class" : "true",
+              "count" : 226
+            }
+          ],
+          "other_predicted_class_doc_count" : 0
+        }
+      ],
+      "other_actual_class_count" : 0
+    }
+  }
+}
+--------------------------------------------------
+<1> The name of the actual class. In this example, there are two actual classes: 
+`true` and `false`.
+<2> The number of documents in the dataset that belong to the actual class.
+<3> The name of the predicted class.
+<4> The number of documents belong to the actual class that are labelled as the 
+predicted class. 
+
+The returned confusion matrix shows us how many datapoints were classified 
```

Comment:
> The returned confusion matrix shows... (actual class does not match predicted class).

I'd suggest moving this first sentence so that it introduces the response example. Otherwise, I have to wait til I get here to understand what I'm looking at.